### DISCUSSION TITLE
Restricting Files Within File Manager

### DISCUSSION THREAD

**Discussion by ajf15cc** [requarks/wiki#2868] [Help / Questions]
Hi,

Apologies if this is a really stupid question but I've been a bit stumped. Is it possible to restrict the files that a user can see within the file manager (in order to upload/add content to a page) to files that they have uploaded themselves; rather than seeing all the files that anyone has uploaded? Even the ability to only see uploads on a page by page or group by group basis would be helpful but at the moment all my users can see everything. As far as I am aware these files can then be viewed on the new page even if the user didn't have rights to view the original page?

If I have misconfigured a file rule or something can you just nudge me in the right direction and I'll give it another look. 

Thanks

Ajay

**Comment by NGPixel** [comment]
Assets are governed by the same page rules you assign to groups.

Check your page rules with the permissions `assets:read`.

**Reply by ajf15cc** [reply]
Hi, Thanks for that. However, when I check the group permissions I have allowed ALL rights to any page that starts with /NTU, however a user within that group (and not in any other groups) is unable to upload files here. A message saying "file upload failed" is seen.
![image](https://user-images.githubusercontent.com/64101895/103231659-3caa8c80-4930-11eb-9db9-873a8ee7ecf0.png)

Global Permissions are also set for that group as per the below:
![image](https://user-images.githubusercontent.com/64101895/103231688-58159780-4930-11eb-9f33-fd27d0da4472.png)

Have you any idea why the file upload is being rejected?

Thanks

**Reply by ajf15cc** [reply]
If it helps the exact error response was {"succeeded":false,"message":"You are not authorized to upload files to this folder."}

**Reply by Serizao** [reply]
Hello, I have the same problem have you find any solution ?

**Reply by ajf15cc** [reply]
Nah, never managed to get it working unfortunately.
